server/sitemap.ts

```diff
@@ -25,9 +26,23 @@ const generateSitemapPage = (
   root: string,
   { loc, lastmod = defaultLastmod, changefreq = "daily", priority }: SitemapPage
 ) => {
+  const firstLvlNav = getFirstLvlNav(loc);
+  let canonicalLoc = loc;
+
+  switch (firstLvlNav) {
+    case "cloud":
+      canonicalLoc = `${canonicalLoc}?scope=cloud`;
+      break;
+    case "enterprise":
+      canonicalLoc = `${canonicalLoc}?scope=enterprise`;
+      break;
+    default:
+      canonicalLoc = loc;
```

Comment:
technically the `default` case is not required because of the variable instantiation on line 30 :)